web_src/css/repo.css

```diff
@@ -654,13 +654,16 @@ td .commit-summary {
 
 .repository.view.issue .comment-list .code-comment {
   border: 1px solid transparent;
-  margin: 0;
+  margin: 0 -4px;
+  padding: 8px;
 }
 
 .repository.view.issue .comment-list .code-comment .comment-header {
   background: transparent;
   border-bottom: 0;
   padding: 0;
+  min-height: auto;
```

Comment:
Maybe the `min-height: 41px` should be removed from `.comment-header`, then no need to add a new `min-height: auto;` here?

Reply:
That min-height is targeting the full-size comment header, but for the inline ones, it seems to use a bit too much space, which is the reason for the auto height there.

<img width="825" height="275" alt="image" src="https://github.com/user-attachments/assets/a54848df-6396-457f-a7a3-bca0287aae0a" />